### PR TITLE
[coverage] Tests for ManufDB + Multiple fixes

### PR DESCRIPTION
**Pull request by gpotter2**
This PR brings several small fixes to improve coverage:
- Add the tests that do not require OpenSSL on appveyor
- Fixes path detection of ManufDB to use the detected Wireshark path.
- Added __repr__ function for manufdb
- Small windows arch tests

### CODE REVIEW

**Comment by gpotter2** [comment]
@guedou I would like you to review this please

File: appveyor.yml [hunk 1]
```diff
@@ -24,18 +24,25 @@ test_script:
   - set PATH="%APPVEYOR_BUILD_FOLDER%;C:\Program Files\Wireshark\;%PATH%"
   
   # Main unit tests
-  - "%PYTHON%\\python -m coverage run -a bin\\UTscapy -f text -t test\\regression.uts -F -K mock_read_routes6_bsd || exit /b 42"
+  - "%PYTHON%\\python -m coverage run --parallel-mode bin\\UTscapy -f text -t test\\regression.uts -F -K mock_read_routes6_bsd || exit /b 42"
```
Comment (guedou) [inline appveyor.yml]: Does `--parallel-mode` improve the speed ?

File: test/mock_windows.uts [hunk 2]
```diff
@@ -147,3 +147,17 @@ emulate_main_input(data)
...
+
+############
+############
++ Windows arch unit tests
+
+
+= show_interfaces
+
+from scapy.arch import *
+show_interfaces()
```
Comment (guedou) [inline test/mock_windows.uts]: Could you check the output of this function ? When we can do it, I like when we mock instead of calling a function. That gives us a better value for code coverage.

File: appveyor.yml (hunk 1 above)
Comment (gpotter2) [inline appveyor.yml]: It doesn't really increase or decrease the speed, but is required to be able to use
`coverage combine`

File: appveyor.yml [hunk 3]
```diff
@@ -24,18 +24,25 @@ test_script:
...
   # Secondary unit tests
   - 'del test\bpf.uts' # Don't bother with BPF regression tests
-  - "for %%t in (test\\*.uts) do (%PYTHON%\\python -m coverage run -a bin\\UTscapy -f text -t %%t -F -K combined_modes_ccm || exit /b 42)"
+  - "for %%t in (test\\*.uts) do (%PYTHON%\\python -m coverage run --parallel-mode bin\\UTscapy -f text -t %%t -F -K combined_modes_ccm || exit /b 42)"
   
   # Contrib unit tests
-  - "for %%t in (scapy\\contrib\\*.uts) do (%PYTHON%\\python -m coverage run -a bin\\UTscapy -f text -t %%t -F -P \"load_contrib(\'%%~nt\')\"  || exit /b 42)"
+  - "for %%t in (scapy\\contrib\\*.uts) do (%PYTHON%\\python -m coverage run --parallel-mode bin\\UTscapy -f text -t %%t -F -P \"load_contrib(\'%%~nt\')\"  || exit /b 42)"
+  
+  # Tls unit tests
```
Comment (guedou) [inline appveyor.yml]: s/Tls/TLS/

File: appveyor.yml (hunk 1 above)
Comment (guedou) [inline appveyor.yml]: OK. I got it. Why did you remove the `-a` flag ? It is used to append test results to the same .coverage file, otherwise the file only contains tests from the last .uts.

Comment (gpotter2) [inline appveyor.yml]: `-a` is the opposite of `--parallel-mode`: if we want to combine .coverage (with the one located in scapy/tools), they musn't be already combined

**Comment by gpotter2** [comment]
@guedou That should do it. Can you restart travis failures ?

**Comment by gpotter2** [comment]
@p-l- @guedou Should ready to merge.

File: scapy/data.py [hunk 4]
```diff
@@ -165,12 +166,12 @@ def load_manuf(filename):
                     lng=shrt
                 else:
                     lng = l[i+2:]
-                manufdb[oui] = shrt,lng
+                manufdb[oui] = shrt, lng
             except Exception,e:
                 log_loading.warning("Couldn't parse one line from [%s] [%r] (%s)" % (filename, l, e))
     except IOError:
         #log_loading.warning("Couldn't open [%s] file" % filename)
```
Comment (guedou) [inline scapy/data.py]: Could you uncomment this warning ?

**Comment by guedou** [comment]
@gpotter2 I am good to merge this PR as soon as you fix the conflict and uncomment L173 in data.py. Thanks !
